integration/hive/src/main/java/org/apache/carbondata/hive/CarbonDictionaryDecodeReadSupport.java

```diff
@@ -231,11 +231,11 @@ private Writable createWritablePrimitive(Object obj, CarbonColumn carbonColumn)
       case LONG:
         return new LongWritable((long) obj);
       case SHORT:
-        return new ShortWritable((Short) obj);
+        return new ShortWritable((short) obj);
```

Comment:
why ？

Reply:
there is merge conflict in this file i will rebase it